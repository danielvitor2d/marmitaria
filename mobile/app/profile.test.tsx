// import { render } from '@testing-library/react-native';
// import Profile from './profile';

// jest.mock("../src/assets/profile.svg", () => 'string qualquer')
// jest.mock("../src/assets/mini_profile.svg", () => 'outra string qualquer')

// describe('test login page', () => {

//   it('snapshot test', () => {
//     const page = render(<Profile />);

//     expect(page).toMatchSnapshot();
//   })

//   it('Testando botÃ£o de voltar', () => {
//     const page = render(<Profile />);

//     expect(page.getByTestId('goBack')).toBeDefined();
//   })

//   it('Testando botÃ£o de ver restaurantes', () => {
//     const page = render(<Profile />);

//     expect(page.getByTestId('seeRestaurants')).toBeDefined();
//   })
// })