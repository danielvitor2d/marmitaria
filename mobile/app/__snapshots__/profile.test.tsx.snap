// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test login page snapshot test 1`] = `
<RCTScrollView>
  <View>
    <View
      style={
        [
          {
            "width": "100%",
          },
          {
            "marginLeft": 16,
          },
        ]
      }
      testID="goBack"
    >
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <Text />
      </View>
    </View>
    <View
      style={
        [
          {
            "flexBasis": "0%",
            "flexGrow": 1,
            "flexShrink": 1,
          },
          {
            "width": "100%",
          },
          {
            "flexDirection": "column",
          },
          {
            "alignItems": "center",
          },
          {
            "marginLeft": -16,
            "marginTop": -16,
          },
        ]
      }
    >
      <Text
        style={
          [
            {
              "fontSize": 24,
              "lineHeight": 32,
            },
            {
              "color": "#A60C0C",
            },
            {
              "fontWeight": "700",
            },
            [
              {
                "marginLeft": 16,
                "marginTop": 16,
              },
            ],
          ]
        }
      >
        Perfil
      </Text>
      <string qualquer
        className="h-32 w-44 border"
        style={
          [
            {
              "marginLeft": 16,
              "marginTop": 16,
            },
          ]
        }
      />
      <View
        style={
          [
            {
              "width": "100%",
            },
            {
              "flexDirection": "column",
            },
            {
              "marginLeft": -8,
              "marginTop": -8,
            },
            {
              "alignItems": "center",
            },
            [
              {
                "marginLeft": 16,
                "marginTop": 16,
              },
            ],
          ]
        }
      >
        <View
          style={
            [
              {
                "width": "66.666667%",
              },
              {
                "marginLeft": -4,
                "marginTop": -4,
              },
              {
                "marginBottom": 20,
              },
              [
                {
                  "marginLeft": 8,
                  "marginTop": 8,
                },
              ],
            ]
          }
        >
          <Text
            style={
              [
                {
                  "fontSize": 16,
                  "lineHeight": 24,
                },
                {
                  "color": "#A60C0C",
                },
                [
                  {
                    "marginLeft": 4,
                    "marginTop": 4,
                  },
                ],
              ]
            }
          >
            Nome
          </Text>
          <TextInput
            placeholder="Ex. João"
            selectionColor="black"
            style={
              [
                {
                  "paddingLeft": 16,
                  "paddingRight": 16,
                },
                {
                  "paddingBottom": 8,
                  "paddingTop": 8,
                },
                {
                  "borderBottomWidth": 1,
                  "borderLeftWidth": 1,
                  "borderRightWidth": 1,
                  "borderTopWidth": 1,
                },
                {
                  "borderBottomColor": "#797979",
                  "borderLeftColor": "#797979",
                  "borderRightColor": "#797979",
                  "borderTopColor": "#797979",
                },
                {
                  "borderBottomLeftRadius": 12,
                  "borderBottomRightRadius": 12,
                  "borderTopLeftRadius": 12,
                  "borderTopRightRadius": 12,
                },
                [
                  {
                    "marginLeft": 4,
                    "marginTop": 4,
                  },
                ],
              ]
            }
          />
        </View>
        <View
          style={
            [
              {
                "width": "66.666667%",
              },
              {
                "marginLeft": -4,
                "marginTop": -4,
              },
              {
                "marginBottom": 20,
              },
              [
                {
                  "marginLeft": 8,
                  "marginTop": 8,
                },
              ],
            ]
          }
        >
          <Text
            style={
              [
                {
                  "fontSize": 16,
                  "lineHeight": 24,
                },
                {
                  "color": "#A60C0C",
                },
                [
                  {
                    "marginLeft": 4,
                    "marginTop": 4,
                  },
                ],
              ]
            }
          >
            Sobrenome
          </Text>
          <TextInput
            placeholder="Ex. Ferreira"
            selectionColor="black"
            style={
              [
                {
                  "paddingLeft": 16,
                  "paddingRight": 16,
                },
                {
                  "paddingBottom": 8,
                  "paddingTop": 8,
                },
                {
                  "borderBottomWidth": 1,
                  "borderLeftWidth": 1,
                  "borderRightWidth": 1,
                  "borderTopWidth": 1,
                },
                {
                  "borderBottomColor": "#797979",
                  "borderLeftColor": "#797979",
                  "borderRightColor": "#797979",
                  "borderTopColor": "#797979",
                },
                {
                  "borderBottomLeftRadius": 12,
                  "borderBottomRightRadius": 12,
                  "borderTopLeftRadius": 12,
                  "borderTopRightRadius": 12,
                },
                [
                  {
                    "marginLeft": 4,
                    "marginTop": 4,
                  },
                ],
              ]
            }
          />
        </View>
        <View
          style={
            [
              {
                "width": "66.666667%",
              },
              {
                "marginLeft": -4,
                "marginTop": -4,
              },
              {
                "marginBottom": 20,
              },
              [
                {
                  "marginLeft": 8,
                  "marginTop": 8,
                },
              ],
            ]
          }
        >
          <Text
            style={
              [
                {
                  "fontSize": 16,
                  "lineHeight": 24,
                },
                {
                  "color": "#A60C0C",
                },
                [
                  {
                    "marginLeft": 4,
                    "marginTop": 4,
                  },
                ],
              ]
            }
          >
            Endereço
          </Text>
          <TextInput
            placeholder="Av. Equador, 556, Centro"
            selectionColor="black"
            style={
              [
                {
                  "paddingLeft": 16,
                  "paddingRight": 16,
                },
                {
                  "paddingBottom": 8,
                  "paddingTop": 8,
                },
                {
                  "borderBottomWidth": 1,
                  "borderLeftWidth": 1,
                  "borderRightWidth": 1,
                  "borderTopWidth": 1,
                },
                {
                  "borderBottomColor": "#797979",
                  "borderLeftColor": "#797979",
                  "borderRightColor": "#797979",
                  "borderTopColor": "#797979",
                },
                {
                  "borderBottomLeftRadius": 12,
                  "borderBottomRightRadius": 12,
                  "borderTopLeftRadius": 12,
                  "borderTopRightRadius": 12,
                },
                [
                  {
                    "marginLeft": 4,
                    "marginTop": 4,
                  },
                ],
              ]
            }
          />
        </View>
      </View>
    </View>
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "alignItems": "center",
          "alignSelf": "center",
          "borderBottomLeftRadius": 12,
          "borderBottomRightRadius": 12,
          "borderTopLeftRadius": 12,
          "borderTopRightRadius": 12,
          "marginTop": 40,
          "opacity": 1,
          "paddingBottom": 8,
          "paddingLeft": 16,
          "paddingRight": 16,
          "paddingTop": 8,
          "width": "66.666667%",
        }
      }
      testID="seeRestaurants"
    >
      <Text
        style={
          [
            {
              "fontSize": 18,
              "lineHeight": 28,
            },
            {
              "color": "#fff",
            },
          ]
        }
      >
        Ver restaurante
      </Text>
    </View>
  </View>
</RCTScrollView>
`;
